dipy/viz/ui.py

```diff
@@ -2161,3 +2163,185 @@ def handle_move_callback(self, i_ren, obj, slider):
         self.move_handle(click_position=click_position)
         i_ren.force_render()
         i_ren.event.abort()  # Stop propagating the event.
+
+
+class ImageHolder(UI):
+    """ A 2D container to hold an image and is of type vtkTexturedActor2D.
```

Comment:
I don't think it's necessary to say what type of actor is used. Just leave it at "A 2D container to hold an image."